resources/js/components/Product/AddToCart.vue

```diff
@@ -64,14 +64,16 @@
                 this.qty = event.target.value
             },
 
-            async add() {
+            async add(count) {
```

Comment:
Aren't we getting any errors when we don't specify a param this way?

Reply:
The usage of the param is checked later, defaulting to `this.qty` when it's not specified

Reply:
So doing `add()` without specifying a param works without errors?

Reply:
Alright; why are the tests failing?

Reply:
I've changed things around slightly as I'm thinking the `$event` is actually getting passed to the function as a param causing it to fail.

It should be extra safe and backwards compatible now :)